marimbabot_planning/launch/marimbabot.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+    <node name="lilypond_to_hitpoint" pkg="marimbabot_planning" type="lilypond_to_hitpoint.cpp"/>
+    <node name="move_to_pose" pkg="marimbabot_planning" type="move_to_pose.cpp"/>
```

Comment:
Same for here. node name defined as "marimba_move" in the move_to_pose.cpp. Would be nice to keep the node names as same.